services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -397,10 +396,9 @@ async def __load_projects(
                 continue
             try:
 
-                with concurrent.futures.ThreadPoolExecutor() as pool:
-                    await asyncio.get_event_loop().run_in_executor(
-                        pool, ProjectAtDB.from_orm, row
-                    )
+                await asyncio.get_event_loop().run_in_executor(
+                    None, ProjectAtDB.from_orm, row
```

Comment:
why did you decide for default executor here and not in the others?

Reply:
Because it did not make sense to have manage the creation and destruction of a ThreadPool when there was already a default one. Less code, less bugs.